package.json

```diff
@@ -260,9 +260,9 @@
     "semantic-release": "17.3.1",
     "shelljs": "0.8.4",
     "strip-ansi": "6.0.0",
-    "tmp-promise": "3.0.2",
     "type-fest": "0.20.2",
-    "typescript": "4.1.3"
+    "typescript": "4.1.3",
+    "tmp-promise": "3.0.2"
```

Comment:
Why this change?

Reply:
No good reason - before `privateCacheDir` was introduced I was using `tmp-promise` to ensure the clone dir was removed on process exit, so it had to move from `devDependencies` to `dependencies`, and after that I moved it back.

Just pushed a commit that restores the same order as master to avoid diff noise!